.buildkite/premerge.steps.yaml

```diff
@@ -8,39 +8,30 @@
 #  note that the setup is already done, so you should not manually adjust anything through the BuildKite interface.
 #
 common: &common
-  timeout_in_minutes: 60 # TODO(ENG-548): reduce timeout once agent-cold-start is optimised.
+  timeout_in_minutes: 60
   retry:
     automatic:
-        # This is designed to trap and retry failures because agent lost connection. Agent exits with -1 in this case.
+      # This is designed to trap and retry failures because agent lost connection. Agent exits with -1 in this case.
       - exit_status: -1
         limit: 3
 
-linux: &linux
+common_builder: &common_builder
+  <<: *common
   agents:
-    - "capable_of_building=graphos-eng"
+    - "agent_count=8"
+    - "capable_of_building=platform"
     - "environment=production"
+    - "machine_type=octa"
```

Comment:
This feels like quite a large machine, especially seeing as Keanu rarely gets built. Might be better to go for smaller machines and less agents per machine.